README.md

```diff
@@ -38,3 +38,6 @@ If you'd like to set a global context for your process, you can do the following
 ```ruby
 contextual_logger.global_context = { service_name: 'test_service' }
 ```
+
+## Contributions
```

Comment:
This seems like a mismatch. Do we want contributions to link to the guidelines (which might also suggest adherence to the COC) and and Code of Conduct section to link to it from the README?

Reply:
Good point, that was an oversight from initial work. I've updated this to link to our contribution guidelines and moved the code of conduct messaging to inside there.